tests/typecheck/managers/querysets/test_as_manager.yml

```diff
@@ -140,6 +140,48 @@
                 class MyModel(models.Model):
                     objects = ModelQuerySet.as_manager()
 
+-   case: includes_custom_queryset_methods_on_unions
+    main: |
+        from myapp.models import MyModel1, MyModel2
+        import sys
+        if sys.version_info <= (3, 9):
```

Comment:
you can't do `type[MyModel]` in python 3.8 and you can't do `MyModel1 | MyModel2` in python3.9

Reply:
Don't bother with the conditional. Use this style in tests:

```python
from typing import Type, Union
kls: Type[Union[MyModel1, MyModel2]] = MyModel1
```

Reply:
ok, done